apps/wpcom-block-editor/src/common/tracking/wpcom-block-picker-search-term-handler.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_picker_no_results`.
+ * Also, it tracks `wpcom_block_picker_no_results` is the searcher doesn't return any result.
+ *
+ * @return {{handler: function, selector: string, type: string}} event object definition.
+ */
+export default () => ( {
+	selector: '.block-editor-inserter__search',
+	type: 'keyup',
+	handler: ( event, target ) => {
```

Comment:
Noting that this will need `debounce` from Lodash to avoid recording every keypress.

When added directly here causes:
```
VM2859 common.min.js:1 Uncaught TypeError: e.mapping.handler is not a function
    at VM2859 common.min.js:1
    at Array.forEach (<anonymous>)
    at HTMLDocument.E (VM2859 common.min.js:1)
```

...so will need some kind of wrapper I suppose.

Reply:
Ahh, screw that. I think I just had wonky lodash import that didn't fail the build so went unnoticed. 🤷‍♂ 

https://github.com/Automattic/wp-calypso/pull/34992